dll/cpl/desk/lang/zh-CN.rc

```diff
@@ -1,16 +1,19 @@
-/*
- * (Simplified Chinese resources)
- * (简体中文资源)
- * by zhangbing 2008 <e_zb@21cn.com, ezb@mail.gywb.cn>
- * Updated: 2009-04-26 Jingjing Fu <jingjingf AT 188.com>
+ /*
```

Comment:
If you are changing the license header, please use the SPDX header:
(For example: https://git.reactos.org/?p=reactos.git;a=blob;f=modules/rostests/rosautotest/CWebService.cpp;hb=bad1c035295455f95ff33045337e8f06996eb7dd )

Reply:
I think the license header I submitted has no problem.